pkg/resource/aggregator.go

```diff
@@ -0,0 +1,215 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
`pkg/resource` feels a bit generic.
I think I would move that to `pkg/license`. So in `main.go` we call `license.NewResourceReporter` instead of `resource.NewLicenseReporter`. WDYT?

Reply:
Looks better.